linera-service/tests/fixtures/evm_example_execute_message.sol

```diff
@@ -20,9 +20,7 @@ contract ExampleExecuteMessage {
     function execute_message(bytes memory input) external {
         uint64 increment = abi.decode(input, (uint64));
         value = value + increment;
-        Linera.opt_MessageId memory opt_message_id = Linera.message_id();
-        require(opt_message_id.has_value);
-        last_message_id = opt_message_id.value;
+        last_message_origin_chain_id = Linera.message_origin_chain_id();
```

Comment:
We could maybe check that the origin of the message is the one we expect.
I can help write this code this weekend if that is a problem.

Reply:
How do we know which one to expect here?